src/palette.ts

```diff
@@ -0,0 +1,27 @@
+type RGBColor = [red: number, green: number, blue: number];
+
+// Muted qualitative colour scheme, colourblind safe, but lacking a clear red or medium blue.
+const palette: RGBColor[] = [
+  [68, 170, 153],
+  [136, 204, 238],
+  [51, 34, 136],
+  [17, 119, 51],
+  [153, 153, 51],
+  [221, 204, 119],
+  [204, 102, 119],
+  [136, 34, 85],
+  [170, 68, 153],
+];
+
+const main: RGBColor = [245, 0, 87]; // Material dark theme secondary
+const secondary: RGBColor = [63, 81, 181]; // Material dark theme primary
+
+function getRGBString(color: RGBColor): string {
+  return `rgb(${color.join(",")})`;
+}
+
+function getRandomPalette(): RGBColor {
```

Comment:
How hard would it be to cycle through the colours instead of picking them randomly? As people are likely to do nearby objects at a similar time this should minimise neighbouring objects having the same colours?

Reply:
I can do it now @ChrisBaidoo 's pr is merged....